go/interpreter/lfvm/interpreter_test.go

```diff
@@ -652,34 +655,37 @@ func TestSteps_StaticContextViolation(t *testing.T) {
 	}
 }
 
-// FIXME: rewrite as static gas check (for all opcodes)
-func TestStepsFailsOnTooLittleGas(t *testing.T) {
-	ctxt := getEmptyContext()
-	instructions := []Instruction{
-		{PUSH1, 0},
-	}
+func TestSteps_FailsWithLessGasThanStaticCost(t *testing.T) {
 
-	ctxt.params = tosca.Parameters{
-		Input:  []byte{},
-		Static: false,
-		Gas:    2,
-		Code:   []byte{0x0},
-	}
-	ctxt.gas = 2
-	ctxt.code = instructions
+	for _, op := range allOpCodes() {
+		t.Run(op.String(), func(t *testing.T) {
+			forEachRevision(t, op, func(t *testing.T, revision tosca.Revision) {
+
+				expectedGas := getStaticGasPrices(revision).get(op)
+
+				ctxt := getEmptyContext()
+				ctxt.params = tosca.Parameters{
+					Input:  []byte{},
+					Static: false,
+					Code:   []byte{0x0},
+				}
```

Comment:
Parameters are not consumed by this test, since operations are not getting executed.
```suggestion
```